src/Apps/IdentityVerification/IdentityVerificationApp/StartIdentityVerification.tsx

```diff
@@ -0,0 +1,168 @@
+import { Box, Button, Sans, Serif } from "@artsy/palette"
+import { StartIdentityVerificationMutation } from "__generated__/StartIdentityVerificationMutation.graphql"
+import * as Schema from "Artsy/Analytics/Schema"
+import { ErrorModal } from "Components/Modal/ErrorModal"
+import React, { useState } from "react"
+import { RelayProp } from "react-relay"
+import { useTracking } from "react-tracking"
+import { commitMutation, graphql } from "relay-runtime"
+import createLogger from "Utils/logger"
+
+const logger = createLogger("StartIdentityVerification.tsx")
+
+interface Props {
+  relay: RelayProp
+  identityVerification: {
+    internalID: string
+  }
+}
+
+export const StartIdentityVerification: React.FC<Props> = ({
+  identityVerification,
+  relay: { environment },
+}) => {
+  const [requesting, setRequesting] = useState(false)
+  const [showErrorModal, setShowErrorModal] = useState(false)
+  const { trackEvent } = useTracking()
+
+  const trackClickedContinueToVerification = () => {
+    trackEvent({
+      context_page_owner_id: identityVerification.internalID,
+      action_type: Schema.ActionType.ClickedContinueToIdVerification,
+      context_page: Schema.PageName.IdentityVerificationPage,
+    })
+  }
+
+  async function startIdentityVerification() {
```

Comment:
This function doesn't seem to use `await`. We should stick with using `async`/`await` together or not using them at all.